central/detection/service/service_impl.go

```diff
@@ -219,44 +220,68 @@ func (s *serviceImpl) runDeployTimeDetect(ctx context.Context, enrichmentContext
 	return s.enrichAndDetect(ctx, enrichmentContext, deployment, policyCategories...)
 }
 
-func getObjectsFromYAML(yamlString string) ([]k8sRuntime.Object, error) {
+func getObjectsFromYAML(yamlString string) ([]k8sRuntime.Object, []string, error) {
```

Comment:
I think we should change name of this function or return a struct with objects and ignore objects. `[]string` tells nothing about what's inside.

Reply:
alternatively 
```suggestion
func getObjectsFromYAML(yamlString string) (objects []k8sRuntime.Object, ignoredObjectsRefs []string, err error) {
```